src/non_markov_localization_main.cpp

```diff
@@ -1598,7 +1615,8 @@ void PlayBagFile(const string& bag_file,
                  bool use_point_constraints,
                  double time_skip,
                  ros::NodeHandle* node_handle,
-                 double* observation_error_ptr) {
+                 double* observation_error_ptr,
+                 char *keyframes_file) {
```

Comment:
`char* keyframes_file`
(The convention used is to group the type together, and separate the name by a space)